Project.toml

```diff
@@ -0,0 +1,10 @@
+name = "julia-ctags"
+uuid = "106e64ca-b22d-11e9-257b-81adddf2616c"
+authors = ["janEbert <janpublicebert@posteo.net>"]
+version = "0.1.0"
```

Comment:
I think we can delete moist of this except the `name` field,
since we won't be registering this